pkgpanda/build/__init__.py

```diff
@@ -410,47 +409,6 @@ def get_docker_id(docker_name):
     return check_output(["docker", "inspect", "-f", "{{ .Id }}", docker_name]).decode('utf-8').strip()
 
 
-def hash_str(s: str):
-    hasher = hashlib.sha1()
```

Comment:
To prevent an import cycle. See https://github.com/dcos/dcos/pull/1138/commits/e4facff15584ab6d8ddfa0adda06c60ff359e6ca